dts/bindings/sensor/st,lsm6dso-common.yaml

```diff
@@ -18,9 +19,98 @@ properties:
       enum:
         - 1 # drdy is generated from INT1
         - 2 # drdy is generated from INT2
-      description: Select DRDY pin number (1 or 2).
+      description: |
+        Select DRDY pin number (1 or 2).
 
         This number represents which of the two interrupt pins
         (INT1 or INT2) the drdy line is attached to. This property is not
         mandatory and if not present it defaults to 1 which is the
         configuration at power-up.
+
+    accel-pm:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Specify the accelerometer power mode.
+        Default is configuration at power-up.
+      enum:
+        - 0 # High Performance mode (default)
+        - 1 # Low/Normal Power mode
+        - 2 # Ultra Low Power mode
+
+    accel-range:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Range in dps. Default is power-up configuration.
+      enum:
+        - 0 # 2g  (0.061 mg/LSB)
+        - 1 # 16g (0.488 mg/LSB)
+        - 2 # 4g  (0.122 mg/LSB)
+        - 3 # 8g  (0.244 mg/LSB)
+
+    accel-odr:
+      type: int
+      required: false
+      default: 0
+      description:
+          Specify the default accelerometer output data rate expressed in samples per second (Hz).
+          Default is power-up configuration.
+      enum:
+        - 0  # Power-Down
+        - 1  # 12.5Hz
+        - 2  # 26Hz
+        - 3  # 52Hz
+        - 4  # 104Hz
+        - 5  # 208Hz
+        - 6  # 417Hz
+        - 7  # 833Hz
+        - 8  # 1667Hz
+        - 9  # 3333Hz
+        - 10 # 6667Hz
+
+    gyro-pm:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Specify the gyrometer power mode.
+        Default is configuration at power-up.
+      enum:
+        - 0 # High Performance mode (default)
+        - 1 # Low/Normal Power mode
+
+    gyro-range:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Range in dps. Default is power-up configuration.
+      enum:
```

Comment:
All of the full-scale ranges should be degrees per seconds and all of the resolutions should be in millidegrees per second.

Reply:
> All of the full-scale ranges should be degrees per seconds and all of the resolutions should be in millidegrees per second. (In the comments)

OK, thanks for catching it!!